mountpoint-s3/scripts/network_performance.json

```diff
@@ -0,0 +1 @@
+network_performance_version_2023-04-06.json
```

Comment:
link to [network_performance_version_2023-04-06.json](https://github.com/awslabs/mountpoint-s3/pull/193/commits/57243f938ad8c6f22bea33060c7ebb07b620825e#diff-1002a893ee786859c7516550feb882001ddac3787d0297d000b83fd8eec89fb4)

Reply:
Don't use a symlink for this, put the version number in the JSON itself so the data is self-describing.